joomla/content.md

```diff
@@ -9,31 +9,45 @@ Joomla is a free and open-source content management system (CMS) for publishing
 # How to use this image
 
 ```console
-$ docker run --name some-%%REPO%% --link some-mysql:mysql -d %%IMAGE%%
+$ docker run --name some-%%REPO%% --network some-network -d %%IMAGE%%
```

Comment:
You've used `joomla-network` in the other examples and `some-network` here -- the latter is more common in other image docs, but I don't have a strong preference.  Did you want them to be consistent?

Reply:
oops, yes... I overlooked that.